tests/unit_tests/f/test_vernier_mod.pf

```diff
@@ -66,7 +67,7 @@ contains
     integer :: ierr
 
     ! Initialise Vernier
-    call vernier_init(this%getMpiCommunicator())
+    call vernier_init(this%getMpiCommunicator(), 'main')
```

Comment:
This should probably have no tag so we are testing both with and without the tag through the Fortran interface
```suggestion
    call vernier_init(this%getMpiCommunicator())
```